### PR TITLE
corrected translation

### PR DESCRIPTION
**Pull request by aazae**
correction of the translation, alignment to keep 80 characters per line

(Hi, first pull request ever, so I hope I'm not out of line. Have a good day !)

### CODE REVIEW

**Comment by danielweck** [comment]
@llemeurfr @panaC would you mind checking this corrected French translation please? :)
Thank you! (sorry, my French is a bit ropey ;)

File: src/resources/information/fr.xhtml [hunk 1]
```diff
@@ -25,63 +25,63 @@
...
+      numériques et les caractéristiques d'accessibilité sont bien conçues.
     </p>
     <p>
       <b>PDF</b> : malgré un manque de flexibilité bien connu (pas d'option de
-      refusion) et l'accessibilité, de nombreux livres électroniques ne sont
-      encore disponibles qu'en format PDF, notamment en les bibliothèques
-      publiques.
+      mise en page fluide) et le manque d'accessibilité, de nombreux livres 
+      électroniques ne sont encore disponibles qu'en format PDF, notamment
+      chez les bibliothèques publiques.
```
Comment (llemeurfr) [inline src/resources/information/fr.xhtml:70]: "chez" -> "au sein de certaines"

File: src/resources/information/fr.xhtml [hunk 2]
```diff
@@ -91,81 +91,81 @@
...
     </p>
 
     <h2>Catalogues OPDS</h2>
 
     <p>
       De nombreuses bibliothèques publiques et librairies adoptent
-      <a href="https://opds.io/">OPDS</a> comme un moyen standard d'exposer leur
-      des catalogues sur le Web.
+      <a href="https://opds.io/">OPDS</a> comme un format standard pour 
+	  afficher leur catalogues sur le Web.
```
Comment (llemeurfr) [inline src/resources/information/fr.xhtml:118]: "leur catalogue".

File: src/resources/information/fr.xhtml [hunk 3]
```diff
@@ -91,81 +91,81 @@
...
     </p>
     <p>
       Grâce à l'OPDS, les bibliothèques et les librairies peuvent également
-      exposer les bibliothèque d'un client, c'est-à-dire les livres
-      électroniques que l'utilisateur a acquis et peuvent immédiatement être
-      téléchargés et lus.
+      afficher la bibliothèque d'un client ou usager, c'est-à-dire les livres
+      électroniques que l'utilisateur a acquis, et permet de les téléchargement
```
Comment (llemeurfr) [inline src/resources/information/fr.xhtml:123]: "de les télécharger"

File: src/resources/information/fr.xhtml [hunk 4]
```diff
@@ -91,81 +91,81 @@
...
-      téléchargés et lus.
+      afficher la bibliothèque d'un client ou usager, c'est-à-dire les livres
+      électroniques que l'utilisateur a acquis, et permet de les téléchargement 
+      et de les lire immédiatement.
     </p>
     <p>
       Thorium peut consulter les catalogues OPDS 1 et 2. Il prend également en
-      charge plusieurs OPDS et peut donc permettre aux utilisateurs d'accéder à
-      des à leur bibliothèque personnelle.
+      charge plusieurs formats d'identifications acceptant OPDS et peut donc
```
Comment (llemeurfr) [inline src/resources/information/fr.xhtml:128]: "formats d'identification associés à OPDS"

File: src/resources/information/fr.xhtml [hunk 5]
```diff
@@ -91,81 +91,81 @@
...
         >, une liste est maintenue ici</a
       >).
     </p>
     <p>
       Les organisations du secteur de l'édition, dans le monde entier, sont
       encouragées à adopter LCP comme un moyen de protéger les livres
-      électroniques contre le partage excessif, tout en offrant les utilisateurs
-      une expérience sans friction et en gardant le contenu correctement
-      accessible. Veuillez
+      électroniques contre le partage excessif, tout en offrant aux
```
Comment (llemeurfr) [inline src/resources/information/fr.xhtml:156]: "électroniques" -> "numériques"

File: src/resources/information/fr.xhtml [hunk 6]
```diff
@@ -91,81 +91,81 @@
...
       >).
     </p>
     <p>
       Les organisations du secteur de l'édition, dans le monde entier, sont
       encouragées à adopter LCP comme un moyen de protéger les livres
-      électroniques contre le partage excessif, tout en offrant les utilisateurs
-      une expérience sans friction et en gardant le contenu correctement
-      accessible. Veuillez
+      électroniques contre le partage excessif, tout en offrant aux 
+      utilisateurs une expérience sans friction et en gardant le contenu
```
Comment (llemeurfr) [inline src/resources/information/fr.xhtml:157]: "en gardant" -> "en conservant"

File: src/resources/information/fr.xhtml [hunk 7]
```diff
@@ -188,31 +188,31 @@
...
     <h2>Qui finance cette application</h2>
 
     <p>
-      L'EDRLab est une organisation à but non lucratif. Notre budget provient
-      essentiellement de nos membres. L'EDRLab a commencé en France, mais compte
-      maintenant 60 membres de Europe, Amérique du Nord, Amérique du Sud et
-      Asie.
+      EDRLab est une organisation à but non lucratif. Notre budget provient
+      essentiellement de nos membres. EDRLab a démarré en France, mais compte
+      maintenant 60 membres d'Europe, d'Amérique du Nord, d'Amérique du Sud et
```
Comment (llemeurfr) [inline src/resources/information/fr.xhtml:205]: "60" -> "65"

File: src/resources/information/fr.xhtml [hunk 8]
```diff
@@ -188,31 +188,31 @@
...
+    <p>Ce projet n'aurait pas vu le jour sans :</p>
     <p>
       1- L'implication financière de nos membres fondateurs : Editis, Hachette
-      Livre, Magrigall, Media-Participations, Cercle de la Librarie, Syndicat
-      national de l'Edition. Ils comprennent que le financement normes ouvertes
-      et open-source est le meilleur moyen de dynamiser un marché innovant pour
-      l'industrie de l'édition.
+      Livre, Madrigall, Média-Participations, Cercle de la Librarie, Syndicat
+      national de l'Edition. Ils comprennent que le financement de normes 
+      ouvertes et open-source est le meilleur moyen de dynamiser un marché
```
Comment (llemeurfr) [inline src/resources/information/fr.xhtml:214]: "normes ouvertes" -> "standards ouverts". 
I know, norms vs standards is a subject for long discussion ..

File: src/resources/information/fr.xhtml [hunk 9]
```diff
@@ -232,18 +232,18 @@
...
 
-    <p><img src="./images/piggy-bank.jpeg" alt="Piggy Bank" /></p>
+    <p><img src="./images/piggy-bank.jpeg" alt="Cochon tirelire" /></p>
     <p>
       Vous pouvez également aider Thorium Reader à devenir une meilleure
-      application
+      application en
       <a href="https://www.edrlab.org/become-a-member/"
-        >de devenir membre de l'EDRLab</a
+        >devenant membre de l'EDRLab</a
```
Comment (llemeurfr) [inline src/resources/information/fr.xhtml:240]: "membre d'EDRLab"

File: src/resources/information/fr.xhtml [hunk 10]
```diff
@@ -232,18 +232,18 @@
...
       >.
     </p>
     <p>
       Vous pouvez également faire un don à l'EDRLab via le
```
Comment (llemeurfr) [inline src/resources/information/fr.xhtml:244]: "à EDRLab"

**Review by llemeurfr — CHANGES_REQUESTED** [review]
Thanks a lot for this first pass, the French text was ugly and I never had the time to clean it. If you don't have the time to make the changes I suggest, we can still merge and I apply these changes as another cleaning pass.

**Comment by danielweck** [comment]
Merging into intermediary EDRLab-managed branch, to fix a few things before the final merge to develop.

File: src/resources/information/fr.xhtml (hunk 10 above)
Comment (danielweck) [reply src/resources/information/fr.xhtml:244]: fixed in other branch.

File: src/resources/information/fr.xhtml (hunk 9 above)
Comment (danielweck) [reply src/resources/information/fr.xhtml:240]: fixed in other branch.

File: src/resources/information/fr.xhtml (hunk 8 above)
Comment (danielweck) [reply src/resources/information/fr.xhtml:214]: fixed in other branch.

File: src/resources/information/fr.xhtml (hunk 7 above)
Comment (danielweck) [reply src/resources/information/fr.xhtml:205]: Fixed in other branch.

File: src/resources/information/fr.xhtml (hunk 6 above)
Comment (danielweck) [reply src/resources/information/fr.xhtml:157]: Fixed in other branch.

File: src/resources/information/fr.xhtml [hunk 11]
```diff
@@ -25,63 +25,63 @@
...
+      <b>Livres audio</b> : Un
       <a href="https://www.w3.org/TR/audiobooks/"
         >format standardisé pour les livres audio</a
       >
       a été conçu par le groupe de travail sur l'édition du W3C en 2020, et
-      EDRLab faisait partie de ce groupe de travail. Le Thorium soutient le
-      <a href="https://www.w3.org/TR/lpf/">conditionné (LPF)</a> format livre
-      audio, qui sera bientôt largement utilisé par les libraires, du moins nous
-      l'espérons.
+      EDRLab faisait partie de celui-ci. Thorium soutient le format de livre
```
[the remark this answers is not in the document]
Comment (danielweck) [reply src/resources/information/fr.xhtml:78]: done.

File: src/resources/information/fr.xhtml (hunk 1 above)
Comment (danielweck) [reply src/resources/information/fr.xhtml:70]: done

File: src/resources/information/fr.xhtml [hunk 12]
```diff
@@ -25,63 +25,63 @@
...
+      livres numériques à mise en page fixe, l'enrichissement des livres 
+      numériques et les caractéristiques d'accessibilité sont bien conçues.
     </p>
     <p>
       <b>PDF</b> : malgré un manque de flexibilité bien connu (pas d'option de
-      refusion) et l'accessibilité, de nombreux livres électroniques ne sont
-      encore disponibles qu'en format PDF, notamment en les bibliothèques
-      publiques.
+      mise en page fluide) et le manque d'accessibilité, de nombreux livres 
+      électroniques ne sont encore disponibles qu'en format PDF, notamment
```
[the remark this answers is not in the document]
Comment (danielweck) [reply src/resources/information/fr.xhtml:69]: fixed

File: src/resources/information/fr.xhtml [hunk 13]
```diff
@@ -25,63 +25,63 @@
...
+      contenu textuel est formaté en (X)HTML 5, il est possible de créer des 
+      livres numériques à mise en page fixe, l'enrichissement des livres 
+      numériques et les caractéristiques d'accessibilité sont bien conçues.
     </p>
     <p>
       <b>PDF</b> : malgré un manque de flexibilité bien connu (pas d'option de
-      refusion) et l'accessibilité, de nombreux livres électroniques ne sont
-      encore disponibles qu'en format PDF, notamment en les bibliothèques
-      publiques.
+      mise en page fluide) et le manque d'accessibilité, de nombreux livres
```
[the remark this answers is not in the document]
Comment (danielweck) [reply src/resources/information/fr.xhtml:68]: done

File: src/resources/information/fr.xhtml [hunk 14]
```diff
@@ -25,63 +25,63 @@
...
     <p>
-      Cette <a href="https://edrlab.org">EDRLab</a> application est en constante
-      et vise à devenir une référence pour l'accès à l'EPUB des publications,
-      des documents PDF, des livres audio et des récits visuels, en option
-      protégé par le <a href="https://edrlab.org/readium-lcp">LCP DRM</a> (DRM
-      signifiant Digital Rights Management).
+      Cette application <a href="https://edrlab.org">EDRLab</a> est en constant
+      développement et vise à devenir une référence pour l'accès des livres 
+      numériques EPUB, des documents PDF, des livres audio et des bandes 
+      dessinées, avec la possibilité de les protéger via le
```
[the remark this answers is not in the document]
Comment (danielweck) [reply src/resources/information/fr.xhtml:31]: ok

File: src/resources/information/fr.xhtml [hunk 15]
```diff
@@ -25,63 +25,63 @@
...
+      EDRLab faisait partie de celui-ci. Thorium soutient le format de livre 
+      audio <a href="https://www.w3.org/TR/lpf/">LPF</a>, qui sera bientôt 
+      largement utilisé par les libraires, du moins nous l'espérons.
     </p>
     <p>
-      <b>Narrations visuelles</b> : EDRLab a conçu un format standard pour les
-      bandes dessinées, manga, webtoons et autres styles d'œuvres numériques
+      <b>Bandes dessinées</b> : EDRLab a conçu un format standard pour les
+      bandes dessinées, mangas, webtoons et autres styles d'œuvres numériques
       basés sur des séquences de images.
```
[the remark this answers is not in the document]
Comment (danielweck) [reply src/resources/information/fr.xhtml:85]: yes

File: src/resources/information/fr.xhtml [hunk 16]
```diff
@@ -25,63 +25,63 @@
...
       <a href="https://www.edrlab.org/open-standards/">Divina</a> est le nom de
```
[the remark this answers is not in the document]
Comment (danielweck) [reply src/resources/information/fr.xhtml:86]: done

**Comment by danielweck** [comment]
Follow-up PR: https://github.com/edrlab/thorium-reader/pull/1610

**Pull request merged**